docs/_entities/account_migration.md

```diff
@@ -0,0 +1,57 @@
+---
+title: AccountMigration
+---
+
+This entity is sent when a person changes their diaspora* ID (e.g. when a user migration from one to another pod happens).
+
+## Properties
+
+| Property    | Type                         | Description                                                                          |
+| ----------- | ---------------------------- | ------------------------------------------------------------------------------------ |
+| `author`    | [diaspora\* ID][diaspora-id] | The diaspora\* ID of the closed account.                                             |
+| `person`    | [Profile][profile]           | New profile of a person                                                              |
+| `signature` | [Signature][signature]       | Signature that validates original and target diaspora* IDs with the new key of person |
+
+### Signature
+
+The string that is being used to generate the signature or verify against it follows the pattern:
+
+~~~
+AccountMigration:#{old_diaspora_id};#{new_diaspora_id}
```

Comment:
This is ruby-syntax, so I'm not sure if we should write it that way in the documentation (because it should be independent from ruby), but also don't have a better idea right now.

For [Magic Signatures](https://diaspora.github.io/diaspora_federation/federation/magicsig.html#signature) we have it described with text. So maybe "`AccountMigration` and the old and the new diaspora\* ID, seperated with `:`"? Or maybe something more abstract like `AccountMigration:[old_diaspora_id]:[new_diaspora_id]`? But really, I don't know how to do this the best, what do you think?

(also it should be a `:` between the IDs, here and below at the example. And I just saw that I had it wrong at my comment above ... sorry)